src/components/PatternPlayground/ScalarParameterControl.tsx

```diff
@@ -44,7 +44,13 @@ export const ScalarParameterControl = memo(function ScalarParameterControl({
   parameters,
   setParameters,
 }: ScalarParameterControlProps) {
+  const min = patternParam.min ? patternParam.min : 0;
```

Comment:
Maybe we can do something like this to make sure that we are always respecting explicitly set falsey min/max/steps:
```suggestion
  const min = typeof patternParam.min === "number" ? patternParam.min : 0;
```